Tests/Packet++Test/TestDefinition.h

```diff
@@ -262,3 +262,6 @@ PTF_TEST_CASE(Asn1EncodingTest);
 // Implemented in LdapTests.cpp
 PTF_TEST_CASE(LdapParsingTest);
 PTF_TEST_CASE(LdapCreationTest);
+
+// Implemented in InfiniBandTests.cpp
+PTF_TEST_CASE(InfiniBandPacketParsing);
```

Comment:
Missing new line at the end of file.
```suggestion
PTF_TEST_CASE(InfiniBandPacketParsing);

```